mods/d2k/fluent/rules.ftl

```diff
@@ -310,7 +310,7 @@ actor-upgrade-hightech =
     Unlocks the Atreides Air Strike superweapon.
 
 actor-deathhand =
-    .name = Death Hand
+    .name = Death Hand Missile
```

Comment:
this name is just for the encyclopedia, and in it you see an image of a missile. This change might be redundant